04/app.js

```diff
@@ -49,6 +57,9 @@ class App extends React.Component {
                     />
                     <input type="submit"/>
                 </form>
+                <input name='searchQuery'
+                    value = {searchQuery}
```

Comment:
Tutaj należało napisać `this.state.searchQuery`, bo nie masz tej zmiennej użytej w destrukturyzacji tj. w linii 46